src/main/msp/msp_protocol_v2_inav.h

```diff
@@ -15,4 +15,6 @@
  * along with INAV.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-//#define MSP2_INAV_STATUS            0x1000
\ No newline at end of file
+//#define MSP2_INAV_STATUS            0x1000
+#define MSP2_INAV_TZ                0x1001  //out message   Gets the TZ offset for the local time (returns: minutes(i16))
```

Comment:
I didn't have time to change this file yet, but this API should belong to `msp_protocol_v2_common.h` file and not INAV-specific.